docs/source/technical/Saga_graphical_window_loggin.md

```diff
@@ -0,0 +1,46 @@
+# Using a graphical window in SAGA
+It is possible to use graphical interface on Saga.
+Login to saga using the `-Y` option
+
+<div style="background-color:silver">
+
+_**PRACTICAL EXERCISE:**_
+
+```bash
+ssh -Y <username>@saga.sigma2.no
+```
+
+Do a simple test that to check that it is working
+
+```bash
+xeyes
+```
+
+type `ctrl+C` to kill xeyes
+
+</div>
+
+You can read more on [SAGA manual: X11 forwarding](https://documentation.sigma2.no/getting_started/create_ssh_keys.html#x11-forwarding)
+
+
+## Concrete example
+Bellow is an example that we using IGV, and is required for the tutorial: [Visualizing assembly and associated reads pileup with IGV](../tutorials/Visualisation_assembly_reads_pileup_IGV.md)
```

Comment:
Below is an example where we are using IGV, which is required for the tutorial:

Reply:
ok